apps/settings/js/panels/screen_lock_passcode/screen_lock_passcode.js

```diff
@@ -1,9 +1,11 @@
-/* global SettingsListener */
+/* global PasscodeHelper */
```

Comment:
This could be removed if we load the script using requirejs. Please check the detail below.